threedi_models_and_simulations/widgets/simulation_wizard.py

```diff
@@ -249,9 +257,12 @@ def handle_item_changed(self, item):
                 self.parent_page.parent_wizard.plugin_dock.communication.show_warn(
                     "Units length should be less than 16 characters!"
                 )
+        # Add item to the substances list
+        self.set_substances_data()
+        self.update_substances()
 
-    def get_substances_data(self):
-        substances = []
+    def set_substances_data(self):
```

Comment:
Please add docstrings (here and for other `SubstancesWidget ` methods) to keep code style consistent.